### DISCUSSION TITLE
sort document listing by editable

### DISCUSSION THREAD

**Discussion by langert** [pimcore/pimcore#14882] [Q&A / Support] [answered]
Hi,
when using `$listing = new Document\Listing();` there is no possibility to sort the result by an editable. Table document_editables is not joined by default. How can I query and sort on document_editables without using the Data Hub Extension e.g.?

**Comment by LeonNoBears** [comment, marked as answer]
The [documentations](https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Working_with_PHP_API.html#page_Access-and-modify-internal-object-list-query) shows an example of a join.

**Reply by langert** [reply]
thx, I only searched the documentation for document listings, but the logic is all same from objects ;-)
